locales/es.yml

```diff
@@ -295,24 +296,24 @@ es:
          comment_spam: "Contiene spam"
          comment_noagree: "No está de acuerdo"
          comment_other: "Otra razón"
-         suspect_word: "Suspect Word"
-         banned_word: "Banned Word"
-         body_count: "Body exceeds max length"
+         suspect_word: "Palabra sospechosa"
+         banned_word: "Palabra prohibida"
+         body_count: "El texto exede el límite permitido"
          trust: "Trust"
          links: "Link"
   modqueue:
     account: "reportes de cuentas"
     actions: Acciones
     all: todos
     all_streams: "Todos los Hilos"
-    notify_edited: '{0} edited comment "{1}"'
-    notify_accepted: '{0} accepted comment "{1}"'
-    notify_rejected: '{0} rejected comment "{1}"'
-    notify_flagged: '{0} flagged comment "{1}"'
-    notify_reset: '{0} reset status of comment "{1}"'
+    notify_edited: '{0} comentarios editados "{1}"'
+    notify_accepted: '{0} comentarios aceptados "{1}"'
+    notify_rejected: '{0} comentarios rechazados "{1}"'
+    notify_flagged: '{0} comentarios reportados "{1}"'
+    notify_reset: '{0} resetear el status de comentarios "{1}"'
```

Comment:
Same here, with context it's:

Kim Gardner edited comment "Blah blah blah"

- notify_edited: '{0} ha editado el comentario "{1}"'
- notify_accepted: '{0} ha aceptado el comentario "{1}"'
- notify_rejected: '{0} ha rechazado el comentario "{1}"'
- notify_flagged: '{0} ha reportado el comentario "{1}"'
- notify_reset: '{0} ha reseteado el status del comentario "{1}"'

Feedback on tense, etc, welcome!

Reply:
agree